posthog/temporal/data_imports/pipelines/stripe/__init__.py

```diff
@@ -254,7 +255,7 @@ def update_request(self, request: Request) -> None:
 
 @dlt.source(max_table_nesting=0)
 def stripe_source(
-    api_key: str, account_id: str, endpoint: str, team_id: int, job_id: str, is_incremental: bool = False
+    api_key: str, account_id: Optional[str], endpoint: str, team_id: int, job_id: str, is_incremental: bool = False
```

Comment:
nit: If you want to save the import line then
```suggestion
    api_key: str, account_id: str | None, endpoint: str, team_id: int, job_id: str, is_incremental: bool = False
```

However, both are currently valid. Really wish Python had stuck to the "There should be one-- and preferably only one --obvious way to do it."